### DISCUSSION TITLE
Adding request size to Traffic Watch?

### DISCUSSION THREAD

**Discussion by krivulcik** [ravendb/ravendb#14386] [Q&A] [answered]
Would it be possible to add a column with request size to Traffic Watch?

The request size would be useful for us while investigating certain performance-related scenarios, especially for POST/PUT requests, bulk inserts, etc.

**Comment by grisha-kotler** [comment, marked as answer]
Hi @krivulcik,
You can track the status of this issue here:
https://issues.hibernatingrhinos.com/issue/RavenDB-18804
